src/cc/CCIncludesMinerFee.cpp

```diff
@@ -0,0 +1,86 @@
+/******************************************************************************
+ * Copyright © 2021 The Komodo Core Developers.                               *
+ *                                                                            *
+ * See the AUTHORS, DEVELOPER-AGREEMENT and LICENSE files at                  *
+ * the top-level directory of this distribution for the individual copyright  *
+ * holder information and the developer policies on copyright and licensing.  *
+ *                                                                            *
+ * Unless otherwise agreed in a custom licensing agreement, no part of the    *
+ * SuperNET software, including this file may be copied, modified, propagated *
+ * or distributed except according to the terms contained in the LICENSE file *
+ *                                                                            *
+ * Removal or modification of this copyright notice is prohibited.            *
+ ******************************************************************************/
+#include "cc/CCIncludesMinerFee.h"
+
+/***
+ * Validate that the contract's conditions have been met
+ * @param eval the return value
+ * @param tx the transaction
+ * @param nIn not used
+ */
+bool IncludesMinerFee::Validate(Eval *eval, const CTransaction &tx, uint32_t nIn)
+{
+    int64_t interest;
+    CAmount total = pcoinsTip->GetValueIn(chainActive.LastTip()->GetHeight(),&interest,tx,chainActive.LastTip()->nTime);
```

Comment:
I saw a comment about this not working for lite clients, as they don't have enough data. Can someone explain how CCs function on lite clients?

Reply:
lite clients are remote clients with no full node installed, so they don't have access to chain objects directly (they need a remote api like electrum protocol) and also don't have access to the full node wallet